### PR TITLE
VA-822 comments unverified

### PR DESCRIPTION
**Pull request by anthonycr**
#### Ticket
[VA-822](https://vimean.atlassian.net/browse/VA-822)

#### Ticket Summary
In order to determine if the current user is verified or not, we need to look at the verified emails sent back in the "emails" field by the API for the current authenticated user on the /me endpoint

#### Implementation Summary
I added in the necessary "emails" hooks to get the list of emails that the user has verified. A non-empty list indicates that the user is verified

#### How to Test
hit the /me endpoint without any filters and see that the ArrayList<Email> is not empty if the user has verified an email.

### CODE REVIEW

**Comment by kevinzetterstrom** [comment]
Is there an accompanied mobile PR? It would be nice to see them in tandem.

File: vimeo-networking/src/main/java/com/vimeo/networking/model/Privacy.java [hunk 1]
```diff
@@ -69,10 +69,10 @@ public String getText() {
             return this.text;
         }
 
-        public static PrivacyValue privacyValueFromString(String text) {
-            if (text != null) {
+        public static PrivacyValue privacyValueFromString(String string) {
+            if (string != null) {
                 for (PrivacyValue privacyValue : PrivacyValue.values()) {
-                    if (text.equalsIgnoreCase(privacyValue.text)) {
+                    if (string.equalsIgnoreCase(privacyValue.text)) {
```
Comment (kevinzetterstrom) [inline vimeo-networking/src/main/java/com/vimeo/networking/model/Privacy.java]: Gonna have some conflicts merging my PR now.. :(

File: vimeo-networking/src/main/java/com/vimeo/networking/model/Privacy.java [hunk 2]
```diff
@@ -85,5 +85,5 @@ public static PrivacyValue privacyValueFromString(String text) {
     public String embed;
     public boolean download;
     public boolean add;
-    public String comments;
+    public PrivacyValue comments;
```
Comment (kevinzetterstrom) [inline vimeo-networking/src/main/java/com/vimeo/networking/model/Privacy.java]: 👍

File: vimeo-networking/src/main/java/com/vimeo/networking/model/Privacy.java (hunk 1 above)
Comment (kevinzetterstrom) [inline vimeo-networking/src/main/java/com/vimeo/networking/model/Privacy.java]: Not a big deal though... I will need to adjust the typeadapter anyway

Comment (anthonycr) [inline vimeo-networking/src/main/java/com/vimeo/networking/model/Privacy.java]: 😭 😢 😿

**Comment by kevinzetterstrom** [comment]
Looks good, merge when ready!

**Pull request merged**